src/gauravk/Assignment_3_prg7/IndexMaxLengthText.java

```diff
@@ -0,0 +1,39 @@
+package gauravk.Assignment_3_prg7;
+
+/* #3
+ * Program 9:  return the index having maximum length from given string array.
+input : {"Maulik", "Krishna", "Technocredits", "Kangan", "Harsh"};
+output : 2
+ */
+import java.util.Scanner;
+
+public class IndexMaxLengthText {
+	String [] names;
+	int userInput() {
+		Scanner sc = new Scanner(System.in);
+		System.out.println("How many words you are planning to analyse?");
```

Comment:
We shuld not take input from the user in service method, we should process the input which is given by the user.